plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -165,6 +179,16 @@ def get_bigquery_credentials(cls, profile_credentials):
             details = profile_credentials.keyfile_json
             return creds.from_service_account_info(details, scopes=cls.SCOPE)
 
+        elif method == BigQueryConnectionMethod.BEARER:
```

Comment:
@jtcohen6 do you have any thoughts / opinions on what we call this auth method?

Reply:
I don't mind `bearer` as long as it reflects what BQ calls this ([right?](https://cloud.google.com/bigquery/docs/authorization)). I prefer it to `oauth-secrets`

Reply:
^ it's actually different than the method described in that link, which is kind of why i _do not_ want to call this bearer! Bearer auth actually feels more to me like the change described in https://github.com/fishtown-analytics/dbt/issues/2802 :D

Reply:
QED. Let's call this something else. `oauth-refresh-token`?